packages/components/src/components/Calculator/calculatorReducer.ts

```diff
@@ -0,0 +1,212 @@
+import React, { Reducer } from "react";
```

Comment:
Note: I've moved much of this functionality to ViewerProvider, so this file now isn't used much.